### PR TITLE
partio: new port

### PR DESCRIPTION
**Pull request by jasonliu--**
#### Description

[Partio](http://partio.us) is an open source C++ library for manipulating a variety of point cloud file formats.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by jasonliu--** [comment]
Pull request #7007 depends on this port

**Comment by cjones051073** [comment]
Can you please add the standard first line that defines the file formatting, see e.g.


https://github.com/macports/macports-ports/blob/master/science/root6/Portfile

Then reindent the file cleanly to match that style.

File: graphics/partio/Portfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+			developed by Walt Disney Animation Studios, \
+			for reading, writing, and manipulating a \
+			variety of standard particle formats (GEO, \
+			BGEO, PTC, PDB, PDA). It allows software \
+			applications to read and write point cloud \
+			formats from a variety of computer graphics \
+			programs and can serve as a way to interchange \
+			particle data between various file formats.
+
+github.setup		wdas partio ${version} v
```
Comment (mf2k) [inline graphics/partio/Portfile:24]: The `github.setup` line should be moved to near the top of the Portfile, just above the current `name` line. Then the `name` and `version` lines become redundant and should be removed.

Comment (jasonliu--) [reply graphics/partio/Portfile]: I've made the requested changes.

**Comment by mf2k** [comment]
The commits needs to be squashed into 1 and follow the [GitHub commit message guidelines](https://trac.macports.org/wiki/CommitMessages). 

This will do:
`partio: new port
`

**Comment by jasonliu--** [comment]
Squashed commits as requested.

**Pull request merged**